attempt_6_animals/simple-reference-animals.squiggle

```diff
@@ -29,16 +10,17 @@ value_farmed_cow_year_in_human_qalys = normal({ p10: -0.2, p90: 0.1 })
 // the thing is, it doesn't seem that unlikely to me
 // that cows do lead net positive lives
 weight_cow = mixture([450 to 1800, 360 to 1100], [1/2,1/2])
+wastage = 0.3 to 0.5 // should really be a beta
 lifetime_cow = (30 to 42) / 12
 calories_cow = mixture(0.8M to 1.4M, (500k to 700k) * weight_cow/1000) 
```

Comment:
Note: Any dist that gets used by more that one later dist, should probably be made into a SampleSet. That will give you more correlations.

Reply:
Yes. It seems like I missed wastage in this case, though.